hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -36,7 +36,7 @@
     }
   ],
   wakeup_list: [
-    { name: "debug_cable_wakeup",
+    { name: "wakeup",
```

Comment:
And this wkup_req?

Reply:
yep, probably better to make them all wkup_req, let me do that.